### PR TITLE
About page with vision mission values

### PR DESCRIPTION
**Pull request by janbaykara**
- [X] Raw content
- [X] Link to page
- [x] Generalised layout

### CODE REVIEW

File: src/pages/about.tsx [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+            </Flex>
+            <Box
+                sx={{
+                    mt: 4,
+                    position: ["relative", "fixed"],
+                    bottom: 0,
+                    left: 0,
+                    padding,
+                    pr: 0,
+                    // pb: [5],
```
Comment (conatus) [inline src/pages/about.tsx]: Remove commenting out please.

File: src/pages/about.tsx [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+                            <Styled.h3>Openness</Styled.h3>
+                            <Styled.p>Common Knowledge practices openness, between ourselves and with others. Open, unfinished work invites collaboration and iteration. We want to show others how to do what we do and keep ourselves accountable.</Styled.p>
+                        </Box>
+                        <Box sx={{ width: ['100%', '50%'], pr: [0, 5] }}>
+                            <Styled.h3>Optimism</Styled.h3>
+                            <Styled.p>Common Knowledge is optimistic about the power of people to collectively overcome oppression and build a better future.</Styled.p>
+                        </Box>
+                        <Box sx={{ width: ['100%', '50%'], pr: [0, 5] }}>
+                            <Styled.h3>Cooperation</Styled.h3>
+                            <Styled.p>As a not-for-profit workers cooperative, we also adhere to the cooperative values of self-help, self-responsibility, democracy, equality, equity and solidarity.</Styled.p>
```
Comment (conatus) [inline src/pages/about.tsx]: Bold values.

File: src/pages/about.tsx [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+import { Theme } from "../components/theme"
+import "@csstools/normalize.css"
+import { Link as GatsbyLink } from 'gatsby';
+
+const padding = [3, 4, 5]
+const minorColumnWidth = ["48%", "38.196601125%"]
+
+const IndexPage = () => (
+    <Theme>
+        <SEO title="Our vision, mission and values - Common Knowledge" />
```
Comment (conatus) [inline src/pages/about.tsx]: Remove "Our" which isn't found anywhere else in the text.

File: src/pages/about.tsx [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+                        Vision
+            </Styled.h3>
+                    <Styled.p sx={{ fontSize: [4] }}>We envision a world of abundance, free from oppression.</Styled.p>
+                    <Styled.p sx={{ fontSize: [4] }}>To get there, we need people who are confident in their capacity to self-organise, empowered by digital technology.</Styled.p>
+                    {/*  */}
+                    <Styled.h3 sx={{ fontWeight: "bold", mt: [5, 5, 5], mb: [1, 2], color: "muted" }}>
+                        Mission
+            </Styled.h3>
+                    <Styled.p sx={{ fontSize: [4] }}>Working directly with grassroots activists, we design digital tools that make radical change possible.</Styled.p>
+                    {/*  */}
```
Comment (conatus) [inline src/pages/about.tsx]: These internal comments seem superfulous.

File: src/pages/about.tsx (hunk 2 above)
Comment (janbaykara) [reply src/pages/about.tsx]: Is that a reflection or a suggested CSS change?

File: src/pages/index.tsx [hunk 5]
```diff
@@ -2,144 +2,86 @@
...
-import "@csstools/normalize.css"
+import GridLayout from '../components/layout';
+import { Fragment } from 'react';
 
-const padding = [3, 4, 5]
-const minorColumnWidth = ["48%", "38.196601125%"]
+const number = { m: 0, color: 'accent' }
 
 const IndexPage = () => (
-  <Theme>
```
Comment (conatus) [inline src/pages/index.tsx]: You can use the `<>` format for a fragment.

File: src/pages/about.tsx (hunk 2 above)
Comment (conatus) [reply src/pages/about.tsx]: Ha! A suggested CSS change.

File: src/pages/about.tsx [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+                </Box>
+                <Box sx={{ width: ['100%', '50%'], pr: [0, 5] }}>
+                    <Styled.h4>Openness</Styled.h4>
+                    <Styled.p>Common Knowledge practices openness, between ourselves and with others. Open, unfinished work invites collaboration and iteration. We want to show others how to do what we do and keep ourselves accountable.</Styled.p>
+                </Box>
+                <Box sx={{ width: ['100%', '50%'], pr: [0, 5] }}>
+                    <Styled.h4>Optimism</Styled.h4>
+                    <Styled.p>Common Knowledge is optimistic about the power of people to collectively overcome oppression and build a better future.</Styled.p>
+                </Box>
+                <Box sx={{ width: ['100%', '50%'], pr: [0, 5] }}>
```
Comment (conatus) [inline src/pages/about.tsx]: Abstract out into a shared variable to avoid repetition and save time.

File: src/pages/index.tsx (hunk 5 above)
Comment (janbaykara) [reply src/pages/index.tsx]: This is not possible when using theme-ui unfortunately! https://github.com/system-ui/theme-ui/issues/195#issuecomment-513816495

File: src/pages/about.tsx (hunk 6 above)
Comment (janbaykara) [reply src/pages/about.tsx]: Done.

**Pull request closed without merging**